numpy/ma/core.py

```diff
@@ -3802,7 +3802,10 @@ def __iadd__(self, other):
         else:
             if m is not nomask:
                 self._mask += m
-        ndarray.__iadd__(self._data, np.where(self._mask, 0, getdata(other)))
+        ndarray.__iadd__(
```

Comment:
Yeah, it doesn't work.
```
In [12]: a = ma.masked_array(matrix(eye(3)))

In [13]: a._data += 1
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-13-e93b03f9a4cb> in <module>()
----> 1 a._data += 1

AttributeError: can't set attribute
```
Apparently the `_data` attribute is not separately useable.